### ISSUE TITLE
Can't point to typescript files for fullRouterPath

### ISSUE THREAD

**Issue by levi** [calasanmarko/trpc-swift#3]
The dynamic import in `cli.ts` doesn't appear to support importing typescript files directly, since the script is evaluated with node. Even when compiling my router with `tsc`, script continues to fail because it appears to be outputting as `require` module format instead of `import`.

What does your setup look like and how are you calling the cli command with your typescript project successfully?

**Comment by calasanmarko** [comment]
Yes, the package doesn't support directly importing TypeScript as of now. I don't have much experience directly parsing TS, but it could be a better direction for the project overall as it would get rid of the dependence on Zod.

Part of the reason for choosing to parse the Zod schemas instead of the types was to allow for a future version to include some sort of Zod-like validation logic in the generated Swift code, but I've come to realize that this is too far out-of-scope.

I'll definitely look into parsing the types in the following days, any input/ideas are appreciated.

Could you provide an example for your `import` issue? The `trpc-swift` package is compiled as an ESM module, and the dynamic import should be able to handle CJS. I've tried the cli with both ESM and CJS projects and it has worked fine.

**Comment by levi** [comment]
I resolved the issue by checking out the project directly and running the cli script with `tsx`. Obviously not a solution for the project, but it allows me to skip the js compilation step. The project required a little bit of modification, as it hardcodes `.js` imports.

**Comment by zshannon** [comment]
Wow @levi you beat me to this by 15 hours ✨ can you share what you did with `tsx`?
